vendor/github.com/goji/httpauth/.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: go
+sudo: false
+
+matrix:
+  include:
+    - go: 1.2
+    - go: 1.3
+    - go: 1.4
+    - go: 1.5
```

Comment:
Go versions earlier than `1.5` seems do not support `vendor`, does it affect the test?